newdle/api.py

```diff
@@ -216,3 +217,14 @@ def update_participant(args, code, participant_code):
         setattr(participant, key, value)
     db.session.commit()
     return ParticipantSchema().jsonify(participant)
+
+
+@api.route('/newdle/<code>/participants', methods=('POST',))
+@allow_anonymous
+@use_args(CreateAnonymousParticipantSchema(), locations=('json',))
+def create_anonymous_participant(args, code):
+    newdle = Newdle.query.filter_by(code=code).first_or_404('Invalid code')
+    participant = Participant(name=args['name'], newdle=newdle)
+    db.session.add(participant)
```

Comment:
I think instead of specifying `newdle` above it'd be cleaner to replace this (unnecessary) line with `newdle.participants.add(participant)`